app/views/sites/editor/categories/_form.html.haml

```diff
@@ -0,0 +1,15 @@
+:ruby
+  if @category.new_record?
```

Comment:
This `if` statement is a bit ugly.
Can you use the polymorphic path such as the following?
```ruby
[:sites, :site, :editor, @post, {site_fqdn: current_site.fqdn}]
```